__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const app = require('../app/app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+const endpoints = [
```

Comment:
I understand the reasoning behind this but one nonexistent path would be enough 🙂

Reply:
What about future 404s for when I get to articles and other endpoints, would it not be easier to have it like this rather than re-writing the test for each one?